cmd/tailscale/cli/serve_v2.go

```diff
@@ -45,16 +45,16 @@ a partial URL (e.g., localhost:3000), or a full URL including a path (e.g., http
 
 EXAMPLES
   - Expose an HTTP server running at 127.0.0.1:3000 in the foreground:
-    $ tailscale %s 3000
+    $ tailscale {{subcmd}} 3000
```

Comment:
This confused me at first because it has the syntax of the `text/template` package but I didn't see any imports or usage of the package. 

Let's instead update the format to be not `%s` but instead `%[1]s` -- the fmt package takes one-indexed formatting directives so you don't have to repeat the same argument again below

Reply:
TIL, thanks!